### DISCUSSION TITLE
Can multiple sub commands be executed in a single labgrid-client call?

### DISCUSSION THREAD

**Discussion by Sriram-JI** [labgrid-project/labgrid#1195] [Q&A] [answered]
Hi, 

       I have a small query regarding exporter configuration & mapping into client place creation , usage.
       Query:
                 I have create exporter cfg using 3 resources.
                 I match it with a place listing all 3 resources lets say console, network, imxusbloader.
                 can i able to start a client & able to access all 3 resources at a time?
                 
Thanks,
Sriram

**Comment by Bastian-Krause** [comment]
What do you mean by "start a client & able to access all 3 resources at a time"? With a single labgrid-client call?

**Reply by Sriram-JI** [reply]
yep with a single labgrid-client call. Ex. _"labgrid-client -p <place_name>"_ . Inwhich place has all 3 resources listed

**Reply by Bastian-Krause** [reply]
A place is exactly that: a collection of resource matches + some optional metadata. Please read 
https://labgrid.readthedocs.io/en/latest/getting_started.html#client
and maybe also https://labgrid.readthedocs.io/en/latest/overview.html#coordinator

**Reply by Sriram-JI** [reply]
Thanks for the reference @Bastian-Krause;
I mean like with a single labgrid-client call how can we access two or more resources which is listed in a place?
Ex: i need to access IMXUSBLoader & USBSerialPort within in the same labgrid-client call. Is that possible?

**Reply by Bastian-Krause** [reply, marked as answer]
No, that's not possible. Either create a shell script with multiple labgrid-client commands or even better, write a Python script using [labgrid as a library](https://labgrid.readthedocs.io/en/latest/usage.html#library) to achieve that.
